src/Native/System.IO.Native/nativeio.cpp

```diff
@@ -7,7 +7,7 @@
 #include "nativeio.h"
 #include <sys/stat.h>
 
```

Comment:
I am not seeing that. Are you using my full change including the change to config.h.in?

My intent was not to use (f)stat64 on Mac OS X, which would eliminate the warning and any need to suppress it.